.github/main.workflow

```diff
@@ -1,4 +1,4 @@
-workflow "lint, test, build" {
```

Comment:
I was meaning to ask, we should update these to the yaml declaration instead; I don't these actions work anymore: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-github-actions